resolwe/test_helpers/test_runner.py

```diff
@@ -393,8 +398,31 @@ def run_tests(self, test_labels, **kwargs):
                 self.tags = tags
                 test_labels = tests
 
+                print("Running with following partial tags: {}".format(', '.join(tags)))
+                print("Running with following partial tests: {}".format(', '.join(tests)))
+
         return super().run_tests(test_labels, **kwargs)
 
+    def _git(self, *args):
```

Comment:
Did you consider using any of the Python git libraries?

I'm not saying you should, just interested in your thoughts 🙂 .